tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -589,6 +593,15 @@ const pauseResumeUpload = (page: Page): Promise<void> => {
   return page.locator(pauseResumeUploadButton).click()
 }
 
+export const navigateMediaFile = async ({ page, navigationType }): Promise<void> => {
+  await Promise.all([
+    page.waitForResponse((resp) => resp.status() === 200 && resp.request().method() === 'GET'),
```

Comment:
there can be many request with GET and 200 response
```suggestion
    page.waitForResponse((resp) => resp.status() === 200 && resp.request().method() === 'GET'),
```